pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -876,7 +876,7 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 	isBlockPVCMap := make(map[string]bool)
 	diskInfo := make(map[string]*containerdisk.DiskInfo)
 	for i, volume := range vmi.Spec.Volumes {
-		if volume.VolumeSource.PersistentVolumeClaim != nil {
+		if volume.VolumeSource.PersistentVolumeClaim != nil || volume.VolumeSource.DataVolume != nil {
```

Comment:
Yeah, I've also thought about doing it that way - but this kind of assumes that the name of the PVC will always match the name of the DataVolume, right? it's true that we assume it elsewhere but here it's implicitly defined so it would be harder to detect this if we ever need to.